htdocs/install/mysql/migration/16.0.0-17.0.0.sql

```diff
@@ -55,6 +55,9 @@ ALTER TABLE llx_user DROP COLUMN idpers3;
 
 -- v17
 
+
+ALTER TABLE llx_product ADD COLUMN toproduce tinyint(4) AFTER tobuy;
```

Comment:
What is the goal / use case ?  If a product is to produce, we can just create a BOM on it.
If goal is just to say it is a manufactured product, we already have the field "nature of product".